libs/@guardian/source-react-components/src/choice-card/theme.ts

```diff
@@ -1,6 +1,25 @@
 import { palette } from '@guardian/source-foundations';
 import { userFeedbackThemeDefault } from '../user-feedback/theme';
 
+export interface ChoiceCardTheme {
+	textUnselected?: string;
+	textSelected?: string;
+	textHover?: string;
+	textError?: string;
+	borderUnselected?: string;
+	borderSelected?: string;
+	borderHover?: string;
+	borderError?: string;
+	backgroundUnselected?: string;
+	backgroundHover?: string;
+	backgroundSelected?: string;
+	backgroundTick?: string;
+}
+
+export type ChoiceCardFullTheme = {
+	[P in keyof ChoiceCardTheme]-?: ChoiceCardTheme[P];
```

Comment:
what is this doing? can't quite get mu head around it

Reply:
This just provides a type with all the new properties without the optional `?` which is used for the function which returns the theme to the functions in styles.ts